### ISSUE TITLE
Image Copyright

### ISSUE THREAD

**Issue by BogdanDogaru** [bbc/simorgh#456]
**Is your feature request related to a problem? Please describe.**
Ensure the copyright holder of non-BBC images is displayed and read correctly by screen readers.

**Describe the solution you'd like**
- Overlay the copyright using text (as opposed to having ImageChef add the copyright as a watermark).
- The visible copyright holder should have a contrast of 4.5:1

**Comment by sareh** [comment]
I suggest investigating using an Image CDNs such as Imgix, which does real-time processing: https://docs.imgix.com/apis/url/watermark 
There is a lot of flexibility with applying watermarks to images here in contrast to the current in-house solution (iChef). Watermarks, Branding and Text can be applied with query parameters. There are also very good image optimisation options: https://docs.imgix.com/best-practices/improving-performance

**Comment by pjlee11** [comment]
@BogdanDogaru can you confirm we currently required added the copyright holder as a HTML element as for the near future the copyright won't be embedding in the image?

**Comment by BogdanDogaru** [comment]
@pjlee11 Yes, we will use the copyright as a HTML element for now, as we do not have an ImageChef replacement yet and none is planned.

**Comment by pjlee11** [comment]
Note from Catharine - _We should check if the screen reader acknowledges a pause between the copyright and the next content (e.g. the caption), if reading all the page content out in one go. If it does not pause, we can should see if there is away around that._

**Comment by pjlee11** [comment]
Following the above note, a proposed solution was to had a full stop with content CSS using ‘::after’ in the same colour as the black background for the displayed text. This solution solved the issues when reading out the page content however it creates an additional element when swiping through element by element. This issues regarding the extra element of `.` can be mitigated by adding `role=text` to the Copyright `<span>`.

The above was the outcome of investigation by @greenc05, below are further details.

We could add a full stop with content CSS using ‘::after’ in the same colour as the black background for the displayed text. Quick demo here on my test page - See the difference between the 2 and 3rd image block on the page.  The 2nd image block has no content css and no pause between the image copyright and the caption, and the 3rd image block has content CSS and a pause…
http://2bitstudio.co.uk/code-tests/article/image-block.html
When testing, it’s a 2 finger swipe down to get VoiceOver on iPhone to read all the content out in one go, also make sure you are using Safari.
During testing it was noticed that swiping left to right through each item reads out ‘period’ as an additional swipe for the full stop. If you add role text that stops VoiceOver iPhone from reading out ‘period’ as an extra swipe.

NOTE: Using role text will need careful testing with VoiceOver Mac with Chrome and VoiceOver iPhone with Chrome, inaddition to the usual testing with Safari and VoiceOver Mac and iPhone (and of course our other AT)

**Comment by greenc05** [comment]
Suggested implementation for the new approach of 'image source' as follows, note that this is untested and requires careful testing as using role text with VoiceOver Mac and Chrome in addition to our usual supported AT

```
<span role="text">
    <span class="gs-u-vh">Image source,</span>
    <span class="copyright copyright__full-stop">Roy McDonald</span>
</span>
```
- This still requires the full stop added after 'Roy McDonald' with content CSS
